docs/spec/spec.md

```diff
@@ -314,69 +280,55 @@ metadata:
-      build: ...
-      revisionTemplate: ... # Deprecated way of expressing the same as template.
-      template:
-        metadata:
-          # Name may be specified here following the same rules as Configuration.
-          # When the name collides, the Route must not be updated to the
-          # conflicting Revision.
-        spec: # serving.knative.dev/v1alpha1.RevisionSpec
-          container: ... # See the Container section below
-          volumes: ... # Optional
-          containerConcurrency: ... # Optional
-          timeoutSeconds: ... # Optional
-          serviceAccountName: ...  # Name of the service account the code should run as
-
-  # Example, only one of "runLatest", "release", "pinned" (DEPRECATED), or "manual" can be set in practice.
-  pinned:
-    revisionName: myservice-00013  # Auto-generated revision name
-    configuration:  # serving.knative.dev/v1alpha1.ConfigurationSpec
-      # +optional. The build resource to instantiate to produce the container.
-      build: ...
-      revisionTemplate: ... # Deprecated way of expressing the same as template.
-      template:
-        spec: # serving.knative.dev/v1alpha1.RevisionSpec
-          container: ... # See the Container section below
-          volumes: ... # Optional
-          containerConcurrency: ... # Optional
-          timeoutSeconds: ... # Optional
-          serviceAccountName: ...  # Name of the service account the code should run as
-
-  # Example, only one of "runLatest", "release", "pinned" (DEPRECATED), or "manual" can be set in practice.
-  release:
+spec:
+  template:
+    metadata:
+      # Name may be specified here following the same rules as Configuration.
+      # When the name collides, the Route must not be updated to the
+      # conflicting Revision.
+    spec:
+      # Service supports the full inline Configuration spec.
+      containers: ... # See the Container section below
+      volumes: ...
+      serviceAccountName: ...
+      containerConcurrency: ...
+      timeoutSeconds: ...
+
+  # Service supports an inline Route spec.
+  # Unlike the Route, use of configurationName is disallowed, and the
+  # omission of revisionName will behave as if configurationName were
+  # the name of the Service-managed Configuration with the spec above.
+  # Also unlike Route, this block is optional.  When omitted, it will
+  # default to a simply 100% block that will run the latest revision
+  # instantiated from the above configuration spec.
+  traffic:
+  - revisionName: ...
+    tag: ...  # +optional. This will cause the Service to have a
+                   # stable "url:" associated with it in the status block.
+    percent: 100  # list percentages must add to 100. 0 is a valid list value
+    latestRevision: true | false  # +optional. Matches whether revisionName is omitted.
+    name: ...  # DEPRECATED, see tag.
+  - ...
+
+
+  # We have DEPRECATED support for several "modes", but prefer the style above.
+  runLatest:  # DEPRECATED
```

Comment:
For all other deprecated fields you collapsed them to the top-level setting rather than listing all subfields. I would like to keep it consistent, and might lean towards collapsing them for now.

Reply:
Done